app/components/SubmitApplication.tsx

```diff
@@ -34,7 +35,11 @@ export const SubmitApplicationComponent: React.FunctionComponent<Props> = ({
     );
     console.log(response);
```

Comment:
we can remove this now

Reply:
This was actually already in the code previously, there's a TODO to handle submission errors.